hr_worked_days_from_timesheet/__openerp__.py

```diff
@@ -20,31 +21,22 @@
 
 {
     'name': 'Worked Days From Timesheet',
-    'version': '1.0',
+    'version': '8.0.1.0',
```

Comment:
Please change the version number according to https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers